client/engine/store/mockstore.go

```diff
@@ -184,6 +199,34 @@ func (ms *MockStore) GetTwoPartyLedger(firstParty types.Address, secondParty typ
 	return ledger, ok
 }
 
+// GetConsensusChannel returns a ConsensusChannel between the two parties if it exists.
+func (ms *MockStore) GetConsensusChannel(firstParty types.Address, secondParty types.Address) (channel *consensus_channel.ConsensusChannel, ok bool) {
+
+	ms.consensusChannels.Range(func(key string, chJSON []byte) bool {
+
+		var ch consensus_channel.ConsensusChannel
+		err := json.Unmarshal(chJSON, &ch)
+
+		if err != nil {
+			return true // channel not found, continue looking
+		}
+
+		participants := ch.Participants()
+		if len(participants) == 2 {
+			// TODO: Should order matter?
```

Comment:
Can we resolve this question before merging?

Reply:
If `GetConsensusChannel` accepts a leader and follower address then participant order does matter since the leader is always the first participant and the follower is the second, so we could remove this TODO.

Reply:
[a4568d5](https://github.com/statechannels/go-nitro/pull/426/commits/a4568d5c4aeece2aef34990afb414d95102706fe)